src/api-engine/api/lib/pki/cryptogen/cryptogen.py

```diff
@@ -0,0 +1,42 @@
+from subprocess import call
+from api.config import CELLO_HOME,FABRIC_TOOL
+
+
+class CryptoGen:
+
+    def __init__(self, name, filepath=CELLO_HOME, cryptogen=FABRIC_TOOL, version="2.2.0"):
+        self.cryptogen = cryptogen + "/cryptogen"
+        self.filepath = filepath
+        self.version = version
+        self.name = name
+
+    def generate(self, output="crypto-config", config="crypto-config.yaml"):
+        try:
+            call([self.cryptogen, "generate", "--output={}/{}/{}".format(self.filepath, self.name, output),
```

Comment:
So this is a cli method?

Reply:
another way，we said at the meeting, we can use sdk request a certificate from the fabric ca.